src/tree/taffy_tree.rs

```diff
@@ -25,6 +25,9 @@ use crate::{compute::compute_flexbox_layout, LayoutFlexboxContainer};
 #[cfg(feature = "grid")]
 use crate::{compute::compute_grid_layout, LayoutGridContainer};
 
+#[cfg(feature = "computed_layout_info")]
+use crate::tree::layout::ComputedLayoutInfo;
```

Comment:
I wonder if we can think of a more specific name for this feature? Because all of Taffy's output is computed layout info... Something like `intermediary_layout_info` or `detailed_layout_info` or something?

Reply:
Sure! `detailed_layout_info` sounds goods. Feels like the name for corresponding objects related to the should change corresponding to the feature too.